### PR TITLE
Release 3.0.0

### PR DESCRIPTION
**Pull request by mradkov**
Iris compatible

Node: `6.0.0`
Compiler: `6.0.0`

### CODE REVIEW

File: docker/aeternity_node_mean16.yaml [hunk 1]
```diff
@@ -19,16 +21,17 @@ keys:
...
 
 chain:
-  persist: true
+  persist: false
   hard_forks:
     "1": 0
     "2": 2
     "3": 4
-    "4": 5
+    "4": 6
```
Comment (marc0olo) [inline docker/aeternity_node_mean16.yaml:29]: Iris is protocol v5 and we can activate directly on block with the first block
```suggestion
  hard_forks:
    "1": 0
    "5": 1
```

File: docker/aeternity_node_mean16.yaml [hunk 2]
```diff
@@ -19,16 +21,17 @@ keys:
...
 
 mining:
   autostart: true
   beneficiary: "ak_2iBPH7HUz3cSDVEUWiHg76MZJ6tZooVNBmmxcgVK6VV8KAE688"
+  beneficiary_reward_delay: 2
```
Comment (marc0olo) [inline docker/aeternity_node_mean16.yaml:34]: do we need that here? just want to know what's the reason for adding this :D

**Review by marc0olo — COMMENTED** [review]
looks generally good to me. lot's of stuff included :D ... just one question and the suggestion to activate Iris correctly

File: docker/aeternity_node_mean16.yaml (hunk 2 above)
Comment (mradkov) [reply docker/aeternity_node_mean16.yaml:34]: I thought it will speed up the tests, but I actually don't have the math to verify the claim.

Comment (marc0olo) [reply docker/aeternity_node_mean16.yaml:32]: at least it doesn't hurt :D

**Review by marc0olo — APPROVED** [review]

**Pull request merged**